README.md

```diff
@@ -91,9 +93,11 @@ TODO
 | dex.service.ports.http.port | int | `8000` |  |
 | dex.service.type | string | `"NodePort"` |  |
 | diracx.csVolumeName | string | `"pv-cs-store"` |  |
+| diracx.manageSQLSchema | bool | `true` |  |
+| diracx.mysqlDatabases[0] | string | `"AuthDB"` |  |
+| diracx.mysqlDatabases[1] | string | `"JobDB"` |  |
+| diracx.mysqlDatabases[2] | string | `"JobLoggingDB"` |  |
```

Comment:
This list is not complete.

Reply:
I think it's good enough for now. FYI: The place to change this would be in `diracx/values.yaml` which would then auto update the documentation.